code/go/internal/validator/folder_item_spec.go

```diff
@@ -105,8 +107,21 @@ func loadItemContent(itemPath, mediaType string) ([]byte, error) {
 		return nil, errors.New("file is empty")
 	}
 
-	switch mediaType {
+	if mediaType == "" {
+		return itemData, nil // no item's schema defined
+	}
+
+	basicMediaType, params, err := mime.ParseMediaType(mediaType)
+	if err != nil {
+		return nil, errors.Wrapf(err, "invalid media type (%s)", mediaType)
+	}
+
+	switch basicMediaType {
 	case "application/x-yaml":
+		if v, _ := params["require-document-dashes"]; v == "true" && !bytes.HasPrefix(itemData, []byte("---\n")) {
```

Comment:
Let's add a TODO or similar comment here referencing https://github.com/elastic/elasticsearch/issues/63247 so we know to revisit this change if the ES issue is resolved on their end.

Reply:
Added!

Reply:
Did you mean to reference https://github.com/elastic/package-spec/pull/54 in your comments (as opposed to https://github.com/elastic/elasticsearch/issues/63247)?

Reply:
We had an offline discussion to leave the reference to https://github.com/elastic/package-spec/pull/54 .